bc_access_menu_allocations/manifest.yml

```
@@ -2,6 +2,6 @@
 name: Allocation
 category: ACCESS
 description: "Allocation\r\n"
-icon: 'fas://magic'
+icon: 'fas://usd'
 url: https://allocations.access.org
```

```suggestion
url: https://allocations.access-ci.org
```

I believe it's `access-ci.org`.

---

These are the links that Andrew provided in the Collaboration Team Running Meeting Notes.

```
Links for ACCESS OnDemand
Operations: https://operations.access.org
Allocation: https://allocations.access.org
Support: https://support.access.org
Metrics: https://metrics.access.org
```

---

We can confirm in today's meeting though.